lib/cuda_color_spinor_field.cu

```diff
@@ -1863,23 +1875,23 @@ namespace quda {
     return out;
   }
 
-//! for deflated solvers:
-  cudaColorSpinorField& cudaColorSpinorField::Eigenvec(const int idx) const {
+//! for composite fields:
+  cudaColorSpinorField& cudaColorSpinorField::Component(const int idx) const {
```

Comment:
Correct, I did not consider more generic case since current implementation was sufficient for deflation methods.